tests/testthat/test-test-threads.R

```diff
@@ -127,42 +135,47 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
     mods_complex <- test_threads(mod_complex, .threads = c(2, 4), .cap_iterations = 100, .mode = "local", .dry_run = TRUE)
 
     # Dont overwrite NBURN if set to 0
-    max_evals <- get_est_options("MAXEVAL|NITER|NBURN", mods_complex)
+    max_evals <- get_est_options(mods_complex)
 
+    # Confirm that estimation method didnt change, and that MAXEVAL/NITER was preserved
```

Comment:
This is no longer confirming "the estimation method didn't change", right? (I think that's okay, but comment should be updated.)

Same comment for line 163.